.github/workflows/codegen-review.yml

```
@@ -0,0 +1,30 @@
+    types: [opened, synchronize]
+    branches: [main, develop]
+
+jobs:
+  codegen-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)